srcs/server/server.hpp

```diff
@@ -86,8 +86,8 @@ class	server {
 		std::string get_network_port(void) const 	{return(this->data.network_port);};
 		std::string get_port(void) const 			{return(this->data.port);};
 		std::string get_password(void) const 		{return(this->data.pass);};
-		user& 	get_user(int i);
-		pollfd&	get_pollfd(int i);
+		user& 	get_user(int i)						{return(this->list_of_users.find(i)->second);};
```

Comment:
I believe there was a reason we had to take them out, but if you tested and it works I´m completely fine

Reply:
Ok, it took me ages to finally test it but you we're right, it's not compiling. I'll revert the changes.